brian2tools/nmlimport/nml.py

```diff
@@ -31,7 +31,7 @@ def validate_morphology(segments):
         for segment in segments:
 
             if segment.parent is not None:
-                if segment.parent.fraction_along != 1:
+                if segment.parent.fraction_along not in [0,1]:
```

Comment:
I don't think we correctly support `fraction_along=0`, it would mean to connect to the proximal end of a segment.

Reply:
Currently I was checking if it is connected proximally [here](https://github.com/brian-team/brian2tools/pull/17/files#diff-b80b7aa670e817379b0298762aa269d5R21) I guess we should remove it then :)